### PR TITLE
adding a EIP draft for ERC1155 asset backed NFT extension

### PR DESCRIPTION
**Pull request by liszechung**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

File: EIPS/eip-5505.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+discussions-to: https://ethereum-magicians.org/t/eip-draft-erc1155-asset-backed-nft-extension/10437
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-08-18
+requires: 1155
+---
+
+## Abstract
+To propose an extension of smart contract interfaces for asset-backed, fractionalized projects using EIP-1155 standard such that total acquisition will become possible. This proposal focuses on In-Real-Life asset, where total acquisition should be able to happen.
```
Comment (Pandapip1) [inline EIPS/eip-5505.md:15]:
```suggestion
To propose an extension of smart contract interfaces for asset-backed, fractionalized projects using the [EIP-1155](./eip-1155.md) standard such that total acquisition will become possible. This proposal focuses on physical assets, where total acquisition should be able to happen.
```
Required: I don't understand the first sentence; it appears to be incomplete. Please fix.
Required: EIP-1155 needs to be linked
Required: Use the correct terminology

File: EIPS/eip-5505.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+## Motivation
+Fractionalized, physical asset projects face difficulty when someone wants to acquire 100% of the asset. For example, if someone wants to acquire the whole asset, he needs to buy all NFT pieces so he will become the 100% owner. However he could not do so as it is publicly visible that someone is trying to perform a total acquisition in an open environment like Ethereum, seller will take advantage to set unreasonable high price which hinders the acquisition. Or in other cases, NFTs are owned by wallets with lost keys, such that the ownership will never be at whole. We need a way to enable potential total acquisition.
+
+## Specification
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119.
+
+EIP-1155 compliant contracts MAY implement this EIP for adding functionalities to support total acquisition.
```
Comment (Pandapip1) [inline EIPS/eip-5505.md:23]:
```suggestion
EIP-1155 compliant contracts MAY implement this EIP for adding functionalities to support total acquisition.

```
Required: formatting

File: EIPS/eip-5505.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+```
```
Comment (Pandapip1) [inline EIPS/eip-5505.md:24]:
```suggestion
```solidity
```
Required: Add language

File: EIPS/eip-5505.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+event ForceSaleClaimed(
+	uint256 qtyBurned,
+	uint256 amountClaimed,
+	address claimer
+)
+```
+
+
+## Rationale
+Native ETH is supported by via WETH (EIP-20).
```
Comment (Pandapip1) [inline EIPS/eip-5505.md:67]:
```suggestion
Native ETH is supported by via wrapped Ether ([EIP-20](./eip-20.md)).
```
Required: Use link
Recommended: Don't use undefined abbreviation

File: EIPS/eip-5505.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+After forcedSale is set, the remaining NFTs metadata should be updated to reflect the NFTs are at most valued at the previously set TWAP price.
+
+## Backwards Compatibility
+Needs discussion.
+
+## Test Cases
+Test cases (TODO)
+
+## Reference Implementation
+Implementation (TODO)
```
Comment (Pandapip1) [inline EIPS/eip-5505.md:77]:
```suggestion
```
Required: Optional sections should be removed if not filled in

**Review by Pandapip1 — CHANGES_REQUESTED** [review]
Required: Run this through a spell checker. I can personally recommend Grammarly: https://www.grammarly.com/grammar-check

**Review by liszechung — COMMENTED** [review]
all updated according to suggestion

File: EIPS/eip-5505.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
+---
+eip: 5505
+title: EIP-1155 asset backed NFT extension
+description: Extends [EIP-1155](./eip-1155.md)  to support crucial operations for asset backed NFTs.
```
Comment (Pandapip1) [inline EIPS/eip-5505.md:4]:
```suggestion
description: Extends EIP-1155 to support crucial operations for asset-backed NFTs
```

**Review by Pandapip1 — CHANGES_REQUESTED** [review]

**Pull request merged**